orchestration/dagster_orchestration/hca_orchestration/tests/environments/test_create_snapshot.yaml

```diff
@@ -0,0 +1,7 @@
+resources:
+  snapshot_config:
+    config:
+      dataset_name: 'hca_prod_20201120_dcp2'
+      snapshot_name: 'hca_prod_20201120_dcp2___20210812_dcp8'
```

Comment:
⛏️ Very minor: We should always use fake names in our fixture data and test config. I tend to be extremely paranoid with such things and want it to be abundantly clear that I'm dealing with fake data when I happen to be debugging a test.

Reply:
Is it okay for me to use a slightly modified version of the above snapshot name: "**hca_prod_20201120_dcp2___20210812_fake**"?

I was trying with a much simpler strings like "fake_snapshot_name" or "hca_prod_20201120_dcp2___fake but that always throws an exception because of the name validation code:
`hca_manage.snapshot.InvalidSnapshotNameException: Snapshot name hca_prod_20201120_dcp2___fake is invalid`

Reply:
ah right, because we validate the snapshot name. Yeah the `_fake` qualifier should be fine.